src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,93 @@
+import Image from "next/image";
+
+const Footer = () => {
+  return (
+    <>
+      <footer className="grid bg-[#C3C9A0] p-4 lg:grid-cols-[2fr_1.2fr_1fr] ">
+        <Flag></Flag>
+        <Contact></Contact>
+        <Email></Email>
+      </footer>
+    </>
+  );
+};
+
+const Flag = () => {
+  return (
+    <div className=" text-center md:ml-6 ">
+      <div className=" flex flex-wrap items-center justify-center p-5 ">
+        <div className=" w-auto overflow-hidden  ">
+          <Image
+            src=""
+            alt=""
+            width="0"
+            height="0"
+            className=" float-left mr-2 h-[50px]  w-[50px]"
+          />
+          <Image
+            src=""
+            alt=""
+            width="0"
+            height="0"
+            className=" float-left h-[50px] w-[50px]"
+          />
+        </div>
+
+        <p className=" my-2 ml-3 max-w-sm text-left text-xs font-normal text-black  ">
+          Starick acknowledges Aboriginal and Torres Strait Islander people as
+          the Traditional Custodians of this country and its waters. We pay
+          respect to their Elders past and present and extend this to all
+          Aboriginal people seeing this message.
+        </p>
+      </div>
+
+      <p className="  font-normal text-black   ">
+        © 2023 Starick. All Rights Reserved. |{" "}
+        <a href="#" className="hover:underline">
+          Privacy & Disclaimer
+        </a>
+      </p>
+    </div>
+  );
+};
+
+const Contact = () => {
+  return (
+    <div className="m-3 text-center ">
+      <p className="min-w-min hover:underline  ">
+        <a href="#">Site map / links to pages here</a>
+      </p>
+      <p className="min-w-min hover:underline  ">
+        <a href="#">Site map / links to pages here</a>
+      </p>{" "}
+      <p className="min-w-min hover:underline  ">
+        <a href="#">Site map / links to pages here</a>
+      </p>{" "}
+      <p className="min-w-min hover:underline  ">
+        <a href="#">Site map / links to pages here</a>
+      </p>
+    </div>
```

Comment:
Ideally we want html tags to describe the type of their content. Since this is a list of links, I'd recommend replacing \<div\> with \<ul\> and replacing \<p\> with \<li\>. 

This could also benefit from better adhering to the DRY (don't repeat yourself) principle. This could be done using the JS "map" function to loop over a JSON object containing the link text and addresses. See [here](https://react.dev/reference/react/Children#children-map) for how to do this.

Reply:
THX a lot,  learned so much. I will fix these problems later~😊